app/views/admin/worldwide_organisations_translations/index.html.erb

```diff
@@ -1,55 +1,82 @@
+  items: secondary_navigation_tabs_items(@worldwide_organisation, request.path)
+} %>
 
-      <% if @worldwide_organisation.non_english_translated_locales.present? %>
-        <table id="worldwide-organisations-translations" class="table table-striped table-bordered">
-          <thead>
-            <tr class="table-header">
-              <th>Locale</th>
-              <th>Actions</th>
-            </tr>
-          </thead>
-          <tbody>
-            <% @worldwide_organisation.non_english_translated_locales.each do |locale| %>
-              <tr>
-                <td class="locale">
-                  <%= link_to locale.native_language_name, edit_admin_worldwide_organisation_translation_path(@worldwide_organisation, locale.code) %> (<%= link_to "view", @worldwide_organisation.public_path(locale: locale.code) %>)
-                </td>
-                <td class="actions">
-                  <%= button_to 'Delete',
-                        admin_worldwide_organisation_translation_path(@worldwide_organisation, locale.code),
-                        method: :delete,
-                        class: "btn btn-danger",
-                        data: { confirm: "Are you sure you want to delete this translation?" } %>
-                </td>
-              </tr>
-            <% end %>
-          </tbody>
-        </table>
-      <% else %>
-        <p class="no-content no-content-bordered">No translations</p>
-      <% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Translations",
+      font_size: "l",
+      margin_bottom: 3
+    } %>
 
-      <nav class="form-actions">
-        <% if @worldwide_organisation.missing_translations.any? %>
-          <%= form_tag admin_worldwide_organisation_translations_path(@worldwide_organisation), class: 'well' do %>
-            <div class="form-group">
-              <%= label_tag :translation_locale, 'Locale' %>
-              <%= select_locale :translation_locale, @worldwide_organisation.missing_translations, class: 'form-control input-md-3' %>
-            </div>
-            <%= submit_tag "Create translation", class: "btn btn-primary" %>
-          <% end %>
-        <% end %>
-      </nav>
+    <% if @worldwide_organisation.non_english_translated_locales.present? %>
+      <div class="govuk-table--with-actions govuk-!-margin-bottom-8">
+        <%= render "govuk_publishing_components/components/table", {
+          rows: @worldwide_organisation.non_english_translated_locales.map do |locale|
+            [
+              {
+                text: locale.native_language_name
+              },
+              {
+                text:
+                  link_to(sanitize("Edit #{tag.span("#{locale.native_language_name}", class: "govuk-visually-hidden")}"), edit_admin_worldwide_organisation_translation_path(@worldwide_organisation, locale.code), class: "govuk-link") +
```

Comment:
Should we also have a view link to maintain functionality with the legacy implementation and align to the organisations translations page?

Reply:
We (Nik & me) decided this is not required on this page since it’s not clear where a “View” link would take you. On further reflection, it does raise the question of whether we need these pages at all though (discussed elsewhere).